src/openfermion/hamiltonians/_mean_field_dwave.py

```diff
@@ -35,13 +35,13 @@
 """
 from __future__ import absolute_import
 
+from openfermion.hamiltonians._hubbard import up, down
```

Comment:
I think perhaps more descriptive names for up and down are possible.  I almost feel like they should also be internal, e.g. _up, _down, so such generic names are not in the name space if someone imports * from hubbard.

Reply:
Addressed